python/dolma/core/parallel.py

```diff
@@ -87,15 +87,15 @@ def __init__(
                 file names will also be the same.
             metadata_prefix (str): The prefix of the metadata files to save. This can be a local path or an
                 S3 path. Metadata output will be created for each file after it is processed. Filenames are
-                checked to verify if a file has been processed and can be skipped unless `ignore_existing` is
+                checked to verify if a file has been processed and can be skipped unless `skip_existing` is
                 set to true.
```

Comment:
"checked to verify if a file has been processed and can be skipped if 'skip_existing' is set to true' ", right?